mon-pix/tests/unit/services/url_test.js

```diff
@@ -200,7 +184,7 @@ module('Unit | Service | url', function (hooks) {
         assert.strictEqual(dataProtectionPolicyUrl, expectedDataProtectionPolicyUrl);
       });
 
-      module('when current language is "en"', function () {
+      module('when current language is not "fr"', function () {
         test('returns the French page URL', function (assert) {
```

Comment:
Ici il y a une logique qui m'échappe sur le nouvel enchaînement :
1. when current language is not "fr"
2. returns the French page URL